### ISSUE TITLE
Add RawOps Operations for TensorFlow frontend

### ISSUE THREAD

**Issue by sreegandh804** [unifyai/ivy#1565]
Add [Metrics functions](https://www.tensorflow.org/api_docs/python/tf/keras/metrics#functions_2) for TensorFlow frontend:
- [ ] binary_accuracy

**Comment by Bobbyorr007** [comment]
- [ ] #1565

**Comment by Bobbyorr007** [comment]
- [ ] #1693

**Comment by kimbring2** [comment]
- [ ] #2278

**Comment by Lancelot03** [comment]
- [ ]  ArgMax #2392

**Comment by Lancelot03** [comment]
- [ ]  argmax #2433

**Comment by anapartu** [comment]
- [ ] #2866

**Comment by AmiraOmar99** [comment]
- [ ] #3217

**Comment by ZoeCD** [comment]
- [ ] #3220

**Comment by rush2406** [comment]
- [ ] #3231

**Comment by rush2406** [comment]
- [ ] #3232

**Comment by ZoeCD** [comment]
- [ ] #3264

**Comment by ZoeCD** [comment]
- [ ] #3265

**Comment by PatricYan** [comment]
- [ ] #3350

**Comment by PatricYan** [comment]
- [ ] #3352

**Comment by rush2406** [comment]
- [ ] #3357

**Comment by rush2406** [comment]
- [ ] #3358

**Comment by rush2406** [comment]
- [ ] #3363

**Comment by PatricYan** [comment]
- [ ] #3367

**Comment by PatricYan** [comment]
- [ ] #3368

**Comment by Lancelot03** [comment]
- [ ]  #3370

**Comment by Lancelot03** [comment]
- [ ]  #3371

**Comment by Lancelot03** [comment]
- [ ]  #3372

**Comment by ZoeCD** [comment]
- [ ] #3387
- [ ] #3388
- [ ] #3391

**Comment by Lancelot03** [comment]
- [ ]  #3392

**Comment by johanjino** [comment]
#3485 
#3484

**Comment by rush2406** [comment]
- [ ] #3519 
- [ ] #3520
- [ ] #3521

**Comment by sakibkuet10** [comment]
- #3667

**Comment by carlacampas** [comment]
- [Concat] #3700

**Comment by carlacampas** [comment]
- [Cross] #3700

**Comment by kurshakuz** [comment]
[ ] #3713

**Comment by kurshakuz** [comment]
- [ ] #3733

**Comment by ahmshalan** [comment]
- [ ] #3811

**Comment by dovaaa** [comment]
- [ ] #3939

**Comment by 15tariqj** [comment]
[] #4018

**Comment by zuruoke** [comment]
- [ ] #4416

**Comment by dovaaa** [comment]
- [ ] #4770

**Comment by kcsayem** [comment]
#4779

**Comment by dovaaa** [comment]
- [ ] #4781

**Comment by dovaaa** [comment]
- [ ] #4784

**Comment by zuruoke** [comment]
#4819

**Comment by kcsayem** [comment]
#4821

**Comment by khimraj** [comment]
- [ ] #4848

**Comment by khimraj** [comment]
- [ ] #4850

**Comment by emekaokoli19** [comment]
#4879

**Comment by emekaokoli19** [comment]
#4880

**Comment by dovaaa** [comment]
- [ ] #4909

**Comment by KareemMAX** [comment]
#4939

**Comment by KareemMAX** [comment]
#4943

**Comment by KareemMAX** [comment]
#4944

**Comment by dovaaa** [comment]
- [ ] #5112

**Comment by KareemMAX** [comment]
#5320

**Comment by KareemMAX** [comment]
#5359

**Comment by KareemMAX** [comment]
#5361

**Comment by KareemMAX** [comment]
#5363

**Comment by KareemMAX** [comment]
#5394

**Comment by KareemMAX** [comment]
#5394

**Comment by KareemMAX** [comment]
#5397

**Comment by dovaaa** [comment]
- [ ]  #5399

**Comment by KareemMAX** [comment]
#5407

**Comment by KareemMAX** [comment]
#5408

**Comment by KareemMAX** [comment]
#5411

**Comment by KareemMAX** [comment]
#5412

**Comment by KareemMAX** [comment]
#5413

**Comment by KareemMAX** [comment]
#5414

**Comment by KareemMAX** [comment]
#5422

**Comment by KareemMAX** [comment]
#5457

**Comment by KareemMAX** [comment]
#5458

**Comment by KareemMAX** [comment]
#5459

**Comment by KareemMAX** [comment]
#5461

**Comment by KareemMAX** [comment]
#5464

**Comment by KareemMAX** [comment]
#5470

**Comment by KareemMAX** [comment]
#5472

**Comment by KareemMAX** [comment]
#5474

**Comment by KareemMAX** [comment]
#5475

**Comment by PaoloCatti** [comment]
- [ ] #5553

**Comment by kurshakuz** [comment]
- [ ] #5696

**Comment by hazem171** [comment]
- [ ] #5875

**Comment by Uma-kanagarajan** [comment]
- [ ] #6145

**Comment by Uma-kanagarajan** [comment]
- [ ] Softplus #6360

**Comment by sdk2k01** [comment]
- [ ] #9069

**Comment by KareemMAX** [comment]
https://github.com/unifyai/ivy/issues/9394

**Comment by hmahmood24** [comment]
- [ ] #9466

**Comment by hmahmood24** [comment]
- [ ] #9471

**Comment by hmahmood24** [comment]
- [ ] #9483

**Comment by KareemMAX** [comment]
#9642

**Comment by KareemMAX** [comment]
https://github.com/unifyai/ivy/issues/9643

**Comment by KareemMAX** [comment]
https://github.com/unifyai/ivy/issues/9644

**Comment by sondertg** [comment]
>[ ] AvgPool #9649

**Comment by AdvancedMicroDev** [comment]
#9932

**Comment by revenge47** [comment]
Reverse #9964

**Comment by abdurrahman0123** [comment]
Exit #10140

**Comment by kaushal07wick** [comment]
#10505

**Comment by kaushal07wick** [comment]
getting this error while testing the Conv2D function @ivy-leaves @ivy-root @ivy-seed 

ERROR [ 99%]
test setup failed
file /ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py, line 3064
  def test_tensorflow_Conv2D(
E       fixture 'x_f_d_df' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, frontend, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, run_around_tests, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:3064

**Comment by soma2000-lang** [comment]
Round #10530

**Comment by soma2000-lang** [comment]
Rint #10530

**Comment by sdk2k01** [comment]
- [ ] #10842

**Comment by shreyjain26** [comment]
#11564

**Comment by Husienvora** [comment]
-[]#11611

**Comment by Husienvora** [comment]
- [ ] #11611

**Comment by Husienvora** [comment]
- [ ] #11617

**Comment by KevinUnadkat** [comment]
#11629

**Comment by KevinUnadkat** [comment]
#11633

**Comment by KevinUnadkat** [comment]
#11635

**Comment by IMvision12** [comment]
- [ ] #11699

**Comment by shreyjain26** [comment]
- [ ] #11776

**Comment by shreyjain26** [comment]
- [ ] #11777

**Comment by rigasweb** [comment]
#11962

**Comment by shreyjain26** [comment]
- [ ] #12011

**Comment by shreyjain26** [comment]
- [ ] #12012

**Comment by KevinUnadkat** [comment]
#12548

**Comment by KevinUnadkat** [comment]
#12555

**Comment by KevinUnadkat** [comment]
#12596

**Comment by KevinUnadkat** [comment]
#12598

**Comment by KevinUnadkat** [comment]
#12599

**Comment by KevinUnadkat** [comment]
#12600

**Comment by fabriziocacicia** [comment]
- [ ] #12755

**Comment by luisgarzac** [comment]
Conj #13417

**Comment by ranahgawad** [comment]
- [] #13453

**Comment by ranahgawad** [comment]
: [ ] #13461

**Comment by AryanSharma9917** [comment]
- [ ] #13580

**Comment by kaushal07wick** [comment]
#13488

**Comment by paulaehab** [comment]
- [ ] #13720

**Comment by jddantes** [comment]
- [ ] #13738

**Comment by paulaehab** [comment]
-[ ] #13756

**Comment by paulaehab** [comment]
- [ ] #13782

**Comment by omerfarukballi** [comment]
Hi @unifyai,

I am trying to add my new issue to make label for LogSoftmax frontend function to here.
Would you add this issue to here: LogSoftmax #13877 https://github.com/unifyai/ivy/issues/13877

Thank you!

**Comment by ranahgawad** [comment]
- [ ] #14147

**Comment by ranahgawad** [comment]
- [ ] #14147

**Comment by ranahgawad** [comment]
- [ ] #14147

**Comment by jddantes** [comment]
- [ ] #13738

**Comment by Enigmafiend** [comment]
> * [ ]  [Imag#17595]

**Comment by Romiest** [comment]
#18976

**Comment by Romiest** [comment]
#18994

**Comment by DrZuse** [comment]
#19216

**Comment by janchilling** [comment]
- [] #19308

**Comment by janchilling** [comment]
- [] #19402

**Comment by MuminAhmadKhan** [comment]
-  [ ]  #19624

**Comment by MuminAhmadKhan** [comment]
- [ ] #19709

**Comment by MuminAhmadKhan** [comment]
Also please mark Xlogy, Xlog1py, Xdivy as done  to avoid confusion.
They have been implemented but not marked as done.

**Comment by josej16** [comment]
- [ ] #20579

**Comment by josej16** [comment]
- [ ] #20964

**Comment by a0m0rajab** [comment]
- [ ] #13242

**Comment by a0m0rajab** [comment]
- [ ] #6548

**Comment by a0m0rajab** [comment]
- [ ] #6547

**Comment by a0m0rajab** [comment]
- [ ] #6546

**Comment by a0m0rajab** [comment]
@MuminAhmadKhan Added the related issues, hopefully, the CI will add this to the list soon: 
![image](https://github.com/unifyai/ivy/assets/18273833/de8275fb-ef51-4ab3-af9f-8b86430d3616)

**Comment by mayfieldmobster** [comment]
- [ ] #21271

**Comment by usayrimrehman** [comment]
#21295

**Comment by usayrimrehman** [comment]
#21296

**Comment by usayrimrehman** [comment]
#21297

**Comment by Lancelot03** [comment]
- [ ] #21331

**Comment by ShoaibMajidDar** [comment]
- [ ] #21392

**Comment by Jubii100** [comment]
- [ ] #21545

**Comment by Seckrel** [comment]
- [ ] #21832

**Comment by MuhammadNizamani** [comment]
- [ ] #22085

**Comment by galaxygummybear** [comment]
#22121

**Comment by galaxygummybear** [comment]
#22161

**Comment by usayrimrehman** [comment]
#22240

**Comment by usayrimrehman** [comment]
#22242

**Comment by usayrimrehman** [comment]
#22243

**Comment by usayrimrehman** [comment]
#22244

**Comment by AbdullahSabry** [comment]
- [ ] #22360

**Comment by galaxygummybear** [comment]
Please close this issue #22161. The softsign function has been added and merged into the upstream branch (#22594).

**Comment by 0StevenH0** [comment]
- [ ] #23067

**Comment by Steven-Foo** [comment]
- [ ] #23614

**Comment by Steven-Foo** [comment]
- [ ] #23615

**Comment by Dofften** [comment]
- [ ] #24157

**Comment by Debdeepghosal** [comment]
#26416

**Comment by phattrann** [comment]
- [ ] #26480

**Comment by Ayo-folashade** [comment]
- [ ] #26720

**Comment by RaniaMansour** [comment]
_[ ]# 26768 (ScatterNd)

**Comment by tagrib** [comment]
- [ ] #27015

**Comment by TalhaKhalil** [comment]
#27215

**Comment by TalhaKhalil** [comment]
#27443
